content/nginx-one/k8s/add-nic.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Connect to NGINX One Console
+toc: true
+weight: 200
+nd-content-type: how-to
+nd-product: NGINX One
+---
+
+This document explains how to connect F5 NGINX Ingress Controller <!-- and F5 NGINX Gateway Fabric -->to F5 NGINX One Console using NGINX Agent.
+Connecting NGINX Ingress Controller to NGINX One Console enables centralized monitoring of all controller instances.
+
+Once connected, you'll see a **read-only** configuration of NGINX Ingress Controller. For each instance, you can review:
+
+- Read-only configuration file
+- SSL/TLS certificates
+- CVEs
+
+## Before you begin
+
+Before connecting NGINX Ingress Controller to NGINX One Console, you need to create a Kubernetes Secret with the data plane key. Use the following command:
+
+```shell
+kubectl create secret generic dataplane-key --from-literal=dataplane.key=<Your Dataplane Key> -n <namespace>
+```
+
+When you create a Kubernetes Secret, use the same namespace where NGINX Ingress Controller is running. 
+If you use `-watch-namespace` or `watch-secret-namespace` arguments with NGINX Ingress Controller, 
```

Comment:
Can we link to the explanations for these args in our NIC docs, like we do for NGINX directives?